zipline/build.gradle.kts

```diff
@@ -47,7 +47,9 @@ val copyTestingJs = tasks.register<Copy>("copyTestingJs") {
 }
 
 kotlin {
-  android()
+  android {
+    publishLibraryVariants("release")
```

Comment:
Note that you **must** be using Kotlin 1.5.30 or newer for single-variant publishing to work.